src/status_im/utils/http.cljs

```diff
@@ -190,7 +190,3 @@
   [url]
   (string/lower-case (apply str (map filter-letters-numbers-and-replace-dot-on-dash (url-host url)))))
 
-(defn replace-port
```

Comment:
`replace-port` only used in chat context, for now moved to utils in `status_im2/contexts/chat`.

Reply:
http will be moved to status-im2, so it's better to keep replace-port here

Reply:
ok cool, I can either add this to that pr or just remove the `replace-port` changes and do that in another pr. I will see what's involved. 
Any idea where http stuff should go in the codebase?

Reply:
utils.http

Reply:
ah okay, thanks @flexsurfer I will sort this!

Reply:
so I removed the http migrations from this pr and will do it in a follow up as I believe it will be a much bigger pr and want to keep things as focused as possible 👍